src/main/java/com/feed_the_beast/ftbu/ranks/Rank.java

```diff
@@ -142,8 +142,9 @@ public void fromJson(JsonElement e)
             {
                 String id = a.get(i).getAsString();
                 char firstChar = id.charAt(0);
-                boolean b = firstChar == '-';
-                permissions.put((firstChar == '-' || firstChar == '+') ? id.substring(1) : id, b);
+                boolean not_allowed = firstChar == '-';
```

Comment:
What's this? Doesn't look like anything is modified. Besides, Ranks are highly WIP, you shouldnt touch em. They are non-functional and will get rewritten anyway